app/code/Magento/MediaGalleryCatalogUi/Test/Mftf/Test/AdminMediaGalleryCatalogUiUsedInProductFilterTest.xml

```diff
@@ -60,11 +59,12 @@
         </actionGroup>
 
         <actionGroup ref="AdminAssertMediaGalleryFilterPlaceHolderGridActionGroup" stepKey="assertFilterApplied">
```

Comment:
Please clear the product grid filter after asserting that the product was found, this may be causing other tests to fail.